### ISSUE TITLE
New All Pages [tasklist]

### ISSUE THREAD

**Issue by xell** [toeverything/AFFiNE#2327]
A brand new All Pages will be designed and implemented.

Main features [internal link](https://toeverything.feishu.cn/wiki/wikcnSRjEP363hXwjLyYgHIcnoe):

```[tasklist]
- [ ] https://github.com/toeverything/AFFiNE/issues/2378
- [x] Filter by simple or complex queries.
- [ ] https://github.com/toeverything/AFFiNE/pull/2356
- [ ] https://github.com/toeverything/AFFiNE/issues/2379
- [ ] https://github.com/toeverything/AFFiNE/issues/2380
- [ ] https://github.com/toeverything/AFFiNE/issues/2386
- [ ] https://github.com/toeverything/AFFiNE/pull/2532
```

Related issues:

```[tasklist]
- [ ] #1897
- [ ] #2305
- [ ] #2147
- [x] #2165 ( duplicated as #2168 )
- [ ] #2153
- [ ] https://github.com/toeverything/AFFiNE/issues/1082
- [ ] https://github.com/toeverything/AFFiNE/issues/1982
- [ ] https://github.com/toeverything/AFFiNE/issues/2695
```

**Comment by Yipei-Operation** [comment]
needs to change time description to: 
Today
Yesterday
Last 7 Days
Last 30 Days
Current Year
Earlier

**Comment by xell** [comment]
Considering that both #2305 and #2695  will be designed and implemented in future release, this issue could be closed in the current milestone.

**Comment by zaoying** [comment]
Will there be some `backward compatibilities` likes *workspace migration* in the future?

I have written some articles on the old version in dev mode, then I pulled code from master branch after few days.

Then I run `yarn dev` again, I got some error hints in the browser, then the articles written on old version disappeared.

I thought the app should considering some feature like `backward compatibilities` during the version upgradation.

People should have choice whether drop the article written on old version app, or transform them into new version app. 

Last but not the least, a version history feature of pages like what `git` means to version controller of code,  could be very useful.
